include/configcat/configcatclient.h

```diff
@@ -111,8 +117,12 @@ class ConfigCatClient {
     inline std::shared_ptr<Hooks> getHooks() { return hooks; }
 
 private:
+    struct MakeSharedEnabler;
+
     ConfigCatClient(const std::string& sdkKey, const ConfigCatOptions& options);
 
+    void closeCore();
```

Comment:
In the Swift SDK we call this function `closeResources`. Please rename this to increase consistency between SDKs.